framework/test/unit/components/select/get-matching-value-test.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from 'chai'
+import { getMatchingValue } from '../../../../lib/components'
+
+describe('getMatchingValue', () => {
+  const options = [
+    { label: 'Label1', value: 'value1' },
+    { label: 'Label2', value: 'value2' },
+    { label: 'Label3', value: 'value3' },
+  ]
+
+  it('should return the same value if it is not a string', () => {
```

Comment:
this test is not needed, as the function does not accept `number` per its signature.

i guess, you could either refactor the function or add support for `number` :)

Reply:
Shit so you found the PR

Reply:
Yeah I just removed it